framme/framme-Guadalajara.md

```diff
@@ -0,0 +1,5 @@
+## Guadalajara :fried_shrimp:
```

Comment:
@scathaig it's a city inside the state of Jalisco in Mexico > https://en.wikipedia.org/wiki/Guadalajara